broker-cli/commands/wallet.js

```diff
@@ -261,62 +293,49 @@ async function networkStatus (args, opts, logger) {
     const client = new BrokerDaemonClient(rpcAddress)
     const { baseSymbolCapacities, counterSymbolCapacities } = await client.walletService.getTradingCapacities({market})
 
+    const baseSymbol = baseSymbolCapacities.symbol.toUpperCase()
+    const counterSymbol = counterSymbolCapacities.symbol.toUpperCase()
+
     const statusTable = new Table({
-      head: ['', `${baseSymbolCapacities.symbol.toUpperCase()} Capacity`, `${counterSymbolCapacities.symbol.toUpperCase()} Capacity`],
-      colWidths: [14, 24, 24],
+      head: ['', `${baseSymbol} Capacity`, `${counterSymbol} Capacity`],
```

Comment:
Removed column widths because Table will auto scale to the values in the columns